pythia6.sh

```diff
@@ -1,20 +1,25 @@
 # a pythia6 recipe based on the one from FairROOT
 package: pythia6
 version: "%(tag_basename)s%(defaults_upper)s"
-tag: "428-alice1"
+tag: "alice/428"
```

Comment:
No, this is not ok. We should use a tag, not a branch in the recipes, because otherwise fixes to pythia will trigger a rebuild for everyone. If you really want to have a branch, please do so in a separate default.

Reply:
I agree that one should use a tag instead of a ranch, but there was a reason why I had to use the branch, which I can't recall in the moment. I will try to find out why I switched to the branch.